from giraffe.exceptions.giraffe_exception import GiraffeException


class TechnicalError(GiraffeException):
    pass
